olive/common/config_utils.py

```diff
@@ -207,7 +207,7 @@ def validate_resource_path(v, values, field):
 
     try:
         v = create_resource_path(v)
-    except ValueError as e:
+    except Exception as e:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

what's the consideration we need catch the base exception?

Reply:
forget to modify it back after test